client/hbs/admin.hbs

```diff
@@ -1,31 +1,36 @@
 <html class="admin">
-	{{#if user}}
-	{{set this "admin" true}}
-	{{set this "title" "Admin Panel"}}
-	{{> head.hbs}}
+{{> head.hbs}}
 
+{{#if user}}
 	<header>
 		<h1>Posts</h1>
-		<a href="/editor"><dttw-icon id="new" class="material-icons">note_add</dttw-icon></a>
+		<a href="/editor">
```

Comment:
Ah.  It's because the beautifier treats `<dttw-icon>` as a block element since it doesn't know better.